libethcore/KeyManager.h

```diff
@@ -33,6 +33,22 @@ namespace eth
 {
 class PasswordUnknown: public Exception {};
 
+enum class PasswordValidation
+{
+	Valid,
+	Empty,
+	TooShort,
+	NotEqual
+};
+
+std::string errorDescription(PasswordValidation _validation);
+
+class PasswordValidator
+{
+public:
+	PasswordValidation check(std::string const& _s1, std::string const& _s2) const;
```

Comment:
it's used in https://github.com/ethereum/alethzero/pull/71 . I wanted all executables to eventually use one password validation